SU2_CFD/include/solvers/CAdjTurbSolver.hpp

```diff
@@ -50,6 +50,7 @@ class CAdjTurbSolver final : public CSolver {
    * \brief Return nodes to allow CSolver::base_nodes to be set.
    */
   inline CVariable* GetBaseClassPointerToNodes() override { return nodes; }
+  inline CVariable* GetBaseClassPointerToNodeInfty() override { return nodes; } //TODO Delete me
```

Comment:
NEMO uses a different way to save the farfield state,  node_infty variable container vs the CharacPrimVar vector in develop.  The above lines and following lines allowed me to pass node_infty to the turbulence solver container.....im sure this is very rudimentary.